test/forms/previousRegistrations/PreviousEuVatNumberFormProviderSpec.scala

```diff
@@ -63,5 +64,11 @@ class PreviousEuVatNumberFormProviderSpec extends StringFieldBehaviours {
       val result = form.bind(Map(fieldName -> invalidEuVatNumber)).apply(fieldName)
       result.errors mustBe Seq(FormError(fieldName, invalidKey, Seq(euVatNumberPattern)))
     }
+
+    "must bind when EU VAT number is lowercase" in {
```

Comment:
Can we also add an assertion to  check the value. We have tested the change of the Regex to allow lower case but nothing asserts that the transform works